server/core/src/storage.ts

```diff
@@ -198,7 +202,7 @@ class TServerStorage implements ServerStorage {
     const _class = txClass(tx)
     const objClass = txObjectClass(tx)
     return await ctx.with('tx', { _class, objClass }, async (ctx) => {
-      if (tx.objectSpace !== core.space.DerivedTx) {
```

Comment:
It should be tx.space, please check why it not.

Reply:
It was my mistake, only here, everywhere DerivedTx is tx space